morpheus/service/vdb/milvus_vector_db_service.py

```diff
@@ -275,14 +280,19 @@ def insert(self, data: list[list] | list[dict], **kwargs: dict[str, typing.Any])
 
         return self._insert_result_to_dict(result=result)
 
-    def insert_dataframe(self, df: typing.Union[cudf.DataFrame, pd.DataFrame], **kwargs: dict[str, typing.Any]) -> dict:
+    def insert_dataframe(self,
+                         df: typing.Union[cudf.DataFrame, pd.DataFrame],
+                         truncate_long_strings: bool = False,
```

Comment:
Dont add this here. It belongs on the parent service or should be specified in the client constructur (which gets it's parameters from `get_resource`)

Reply:
I removed it entirely